### PR TITLE
Image in instance entities

### PR DESCRIPTION
**Pull request by vetinari**
* add image_id, image_name and image_date every 15 minutes to the instance entity
* use latest python image to satisfy depencency for setuptools

### CODE REVIEW

**Comment by vetinari** [comment]
:+1:

File: zmon_aws_agent/aws.py [hunk 1]
```diff
@@ -299,10 +298,10 @@ def get_running_apps(region, existing_entities=None):
 
             result.append(ins)
 
-        if (now.minute % 15) == 2:
+        if now.minute == 10:
```
Comment (mohabusama) [inline zmon_aws_agent/aws.py]: I guess this could be an issue if the agent duration missed the 10th minute.

**Comment by mohabusama** [comment]
👍

**Comment by vetinari** [comment]
:+1:

**Pull request merged**